.github/workflows/codeql-analysis.yml

```
@@ -38,11 +38,11 @@ jobs:
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v2
+      uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3.6.0
```

Latest [suggested version](https://github.com/actions/checkout#usage) is just v4. Should do... and will keep updating until v4 is no longer updated (and v5 becomes the main version to follow).

```suggestion
      uses: actions/checkout@v4
```

---

Can a link to an issue be provided? I'm trying to understand what is happening here in light of our upcoming changes to integrate the MS signing policy and our integration with jreleaser

---

This is just a GitHub Action version update for CI, it's separate to any jreleaser efforts.